src/program.js

```diff
@@ -114,6 +115,11 @@ export class Program {
         throw error;
       }
     }
+
+    if (argv.verbose) {
```

Comment:
Nope, this wasn't causing any error at all, it was something completely different.